code/modules/mining/mine_items.dm

```diff
@@ -454,6 +454,12 @@
 	. = ..()
 	T.air_update_turf(1)
 
+//Inivisible, indestructible fans
+/obj/structure/fans/tiny/invisible
```

Comment:
This is better than mappers changing variables they shouldn't, and i see no need to create another type to do exactly the same thing.